src/core/components/InstallButton/index.js

```diff
@@ -72,6 +84,32 @@ export class InstallButtonBase extends React.Component {
           {i18n.gettext('Install Theme')}
         </Button>
       );
+    } else if (addon.type === ADDON_TYPE_OPENSEARCH) {
```

Comment:
If `_window.external.AddSearchProvider` is not defined, the button will do nothing which isn't very helpful. Could we disable the button instead or render an error message?